src/qt/sendcoinsdialog.cpp

```diff
@@ -70,13 +70,20 @@ SendCoinsDialog::SendCoinsDialog(const PlatformStyle *_platformStyle, QWidget *p
 
     // Dash specific
     QSettings settings;
-    if (!settings.contains("bUseDarkSend"))
-        settings.setValue("bUseDarkSend", false);
-    if (!settings.contains("bUseInstantX"))
-        settings.setValue("bUseInstantX", false);
-
-    bool fUsePrivateSend = settings.value("bUseDarkSend").toBool();
-    bool fUseInstantSend = settings.value("bUseInstantX").toBool();
+    //TODO remove Darksend sometime after 0.14.1
+    if (settings.contains("bUseDarkSend"))
+        settings.setValue("bUsePrivateSend", settings.value("bUseDarkSend").toBool());
```

Comment:
should add `settings.remove("bUseDarkSend");` (and wrap `if` contents with `{}`)